lib/Doctrine/ruleset.xml

```diff
@@ -174,8 +174,14 @@
                     @ODM\,
                     @PHPCR\,
                 "/>
-                <element value="@param"/>
-                <element value="@return"/>
+                <element value="
+                    @param,
+                    @psalm-param
```

Comment:
trailing commas if possible, please

Reply:
Makes sense both. Could you commit it here or suggest a patch? I did not clone the project - just created the patch from github directly.
Otherwise I'll clone it tomorrow.